diff --git a/main.py b/main.py
index d8ca7be..0d9f14b 100644
--- a/main.py
+++ b/main.py
@@ -68,7 +68,7 @@ class Voice:
     if platform in ('linux', 'darwin'):
         espeak_binary = 'espeak'
         mbrola_binary = 'mbrola'
-        mbrola_voices_folder = "/usr/share/mbrola"
+        mbrola_voices_folder = "data/voices/mbrola"
     elif platform == 'win32':
         # If the path has spaces it needs to be enclosed in double quotes.
         espeak_binary = '"C:\\Program Files (x86)\\eSpeak\\command_line\\espeak"'
